client/me/purchases/manage-purchase/style.scss

```diff
@@ -109,7 +109,7 @@
 	}
 
 	.is-premium & {
-		border-bottom: 2px solid $alert-green;
```

Comment:
i'm not sure this one should be success. isn't this a brand color? cc @drw158

Reply:
good catch, this should be --color-plan-premium it looks like

Reply:
True 👍  will be covered with #30337